package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "devicon",
-  "version": "2.15.0",
+  "version": "2.15.1",
```

Comment:
You should reverse all the changes in `package.json` since they are not relevant for icon requests